src/components/CheckIn/AddPieceModal.js

```diff
@@ -103,11 +157,12 @@ class AddPieceModal extends Component {
             <Col xs>
               <Field
                 component={Select}
-                dataOptions={locations}
+                dataOptions={[EMPTY_OPTION, ...locations]}
```

Comment:
`EMPTY_OPTION` is required only in case when location is not required (to let user leave it empty). In case the field is required it should be handled through `placeholder=' '` prop.

Reply:
plus this field can be required or not

Reply:
should it be removed now? as you moved it to object